doc/ikfast/ikfast_tutorial.rst

```diff
@@ -58,17 +59,19 @@ You should *not* have mpmath installed: ::
 
 MoveIt! IKFast Installation
 ---------------------------
-Install the MoveIt! IKFast package either from debs or from source.
+Install the MoveIt! IKFast package either from Debian or from source.
```

Comment:
Keep this as is. .deb is the file format of Debian packages, which are also used by Ubuntu and probably other distros.

Reply:
debs is an abbreviation. Debian packages is more clear. It is also the preferred language in the ROS tutorials. http://wiki.ros.org/melodic/Installation/Ubuntu